addons/sales_team/migrations/8.0.1.0/post-migration.py

```diff
@@ -20,12 +20,19 @@
 #
 ##############################################################################
 
+import logging
 from openerp import pooler, SUPERUSER_ID
 from openerp.openupgrade import openupgrade, openupgrade_80
 
+logger = logging.getLogger('OpenUpgrade.sales_team')
+
 
 @openupgrade.migrate(no_version=True)
```

Comment:
Better to remove `no_version` and then the scripts are not executed on module install (which is your case). As you were the original author of the module, maybe you had a reason for putting that.